cmd/root.go

```diff
@@ -427,11 +447,10 @@ func initializeConfig(cmd *cobra.Command, configFile string) error {
 	v.SetConfigType("toml")
 	v.AddConfigPath(filepath.Dir(configFile))
 
-	var err error
-	if err = v.ReadInConfig(); err != nil {
-		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			return err
-		}
+	err := v.ReadInConfig()
+	var notFoundErr viper.ConfigFileNotFoundError
+	if err != nil && !errors.As(err, &notFoundErr) {
+		return err
 	}
```

Comment:
If you only check the type, you can use this


```suggestion
	if err != nil && !errors.As(err, &viper.ConfigFileNotFoundError{}) {
		return err
 	}
```

or

```suggestion
	if err != nil && !errors.As(err, new(viper.ConfigFileNotFoundError)) {
		return err
 	}
```